deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
> If packagers mess this up, they'll get an undefined symbol message and if they grep the code base they'll find this file, so I'm not super concerned.

packagers are busy people too

> Doesn't the version bump force a rebuild?

ah, i missed that. i still think it's better to do this the "right way" (with dlopen) rather than the atypical way (with symbol aliasing)